bfex/components/scrapper/run.py

```diff
@@ -0,0 +1,20 @@
+from scrapper import *
+
+# this should be done with a real service but
+# I do not know how that will work yet
+data = json.load(open(r'../../../../faculty.json'))
```

Comment:
This data will be available in elastic search under the faculty index. But you wont need it if we implement the scraper as designed. It shouldn't know about the faculty at all. Only urls.

Reply:
yes but currently this is the only way to get any urls